### DISCUSSION TITLE
Latest release -> 100% CPU usage on Unraid & docker.img corrupted

### DISCUSSION THREAD

**Discussion by el-fredo** [immich-app/immich#2421] [General]
Yesterday I did the Update to V1.55.0 on my Unraid Homeserver. Since then, I am having a lot of troubles. The CPU usage is constantly at 100% (all four cores) and my other docker-containers are not reachable.

When I "shut down" the Immich stack, everything is fine again.

But this morning that didn't help either and I had to delete the docker.img and set up Docker again. 

Is there anyone else here running Immich on Unraid and observing similar issues?

**Comment by Allram** [comment]
Works fine for me with Unraid. But i guess my system is quite beefy 😅

What happens if you turn off machine learning?

**Reply by el-fredo** [reply]
Turning off machine learning did not improve anything. Processor load still at 100% on all four cores. I also stopped all containers and restartet them one by one. Everything went fine until I restartet the Immich stack.

My home server isn't very powerful, but I've never had such a high load. So far (before the last update) Immich ran flawlessly.

I had to have my docker.img rebuilt twice now because nothing worked any more. I now switched to the Immich version from the Unraid Appstore and I also use my external PostgreSQL database for it now. Much cleaner and safer, I think.

**Reply by Allram** [reply]
If you have to rebuild your Docker image something else if wrong. How much free space do you have on your Docker image?

**Reply by el-fredo** [reply]
Yes, I had to. It was corrupted. Even restarting docker or the whole server didn't solve the problem.

Free space was an issue at the beginnig, because the Immich stack uses a lot of space. So I constantly increased it up to 80 GiB where about 50 GiB are in use. So that's fine.

Maybe something went wrong while pulling the update... I don't know. As mentiones, I am now using the version from the Unraid community app store. I think this is a bit "cleaner". I can specify the docker network, it uses /appdata which I can easily backup and I don't need docker compose or docker folder plugins any more.

I have also learned, that using an external PostgreSQL database & Redis is the way to go. Immich is even faster now :-)
